setup.py

```diff
@@ -85,8 +85,9 @@
 install_requires = [
     'Flask-BabelEx>=0.9.3',
     'Flask-Breadcrumbs>=0.3.0',
+    'Flask-Login>=0.4.0',
     'Flask-OAuthlib>=0.9.3',
-    'Flask-Security>=1.7.5',
+    'Flask-Security-Fork>=1.8.0',
```

Comment:
I removed many, but not all. T_T
```
16:38 $ git grep flask_security
invenio_oauthclient/utils.py:from flask_security.confirmable import requires_confirmation
invenio_oauthclient/utils.py:from flask_security.registerable import register_user
tests/test_handlers.py:from flask_security.confirmable import _security
tests/test_utils.py:from flask_security.confirmable import _security
```

I still have one test failing, but no idea why.
It looks like that after `login_user()` the `g.identity` is still `AnonymousIdentity`. 

Any idea? :)